invenio/modules/formatter/format_elements/bfe_edit_record.py

```diff
@@ -19,16 +19,18 @@
 
 """BibFormat element to print a link to BibEdit."""
 
-from invenio.utils.url import create_html_link
 from invenio.base.i18n import gettext_set_language
 from invenio.config import CFG_BASE_URL, CFG_SITE_RECORD
 from invenio.legacy.bibedit.utils import user_can_edit_record_collection
+from invenio.utils.url import create_html_link
 
 
-def format_element(bfo, style):
+def format_element(bfo, style, html_class, link_label="Edit This Record"):
     """Print a link to BibEdit, if authorization is granted.
 
     :param style: the CSS style to be applied to the link.
+    :param html_class: the class attribute to be applied to the link.
+    :param link_label: Localized link label. Default = "Edit this Record" (or translated variant).
     """
     _ = gettext_set_language(bfo.lang)
```

Comment:
if you use https://github.com/inveniosoftware/invenio/pull/3020/files#r28440182 then remove this line